### PR TITLE
Fixed a few small style issues

### PR DESCRIPTION
**Pull request by bourquenoud**
Fixed a few small issues, like unused variables or redundant code. This cleans up the code a little.

### CODE REVIEW

File: driver/razeraccessory_driver.c [hunk 1]
```diff
@@ -884,10 +880,10 @@ static ssize_t razer_attr_write_matrix_custom_frame(struct device *dev, struct d
...
         }
 
-        row_id = buf[offset++];
-        start_col = buf[offset++];
-        stop_col = buf[offset++];
-        row_length = ((stop_col+1) - start_col) * 3;
+        unsigned char row_id = buf[offset++];
+        unsigned char start_col = buf[offset++];
+        unsigned char stop_col = buf[offset++];
+        unsigned char row_length = ((stop_col+1) - start_col) * 3;
```
Comment (z3ntu) [inline driver/razeraccessory_driver.c]: Generally variables in Linux kernel are declared on top, but from what I find this seems to be allowed now with C11 kernel target https://lwn.net/Articles/885941/ , at least in loops

File: driver/razercommon.c [hunk 2]
```diff
@@ -86,7 +86,7 @@ int razer_get_usb_response(struct usb_device *usb_dev, uint report_index, struct
 
     // Send the request to the device.
     // TODO look to see if index needs to be different for the request and the response
-    retval = razer_send_control_msg(usb_dev, request_report, report_index, wait_min, wait_max);
+    razer_send_control_msg(usb_dev, request_report, report_index, wait_min, wait_max);
```
Comment (z3ntu) [inline driver/razercommon.c]: Perhaps we should add some error handling instead... Not good to ignore errors

File: driver/razeraccessory_driver.c (hunk 1 above)
Comment (bourquenoud) [reply driver/razeraccessory_driver.c]: After thinking about it, I don't think moving the variables down provides anything useful, and I think it actually decreases readability. I'll revert this change.

File: driver/razercommon.c (hunk 2 above)
Comment (bourquenoud) [reply driver/razercommon.c]: Indeed, I'll revert the change and try to add error handling in a different PR

**Review by z3ntu — APPROVED** [review]
Changed some things around but LGTM now

**Pull request merged**